src/sidebar/components/test/NotebookView-test.js

```diff
@@ -134,6 +145,41 @@ describe('NotebookView', () => {
     assert.isTrue(wrapper.find('NotebookFilters').exists());
   });
 
+  describe('synchronization of annotations', () => {
+    beforeEach(() => {
+      fakeStore.focusedGroup.returns({ id: 'hallothere', name: 'Hallo' });
+      fakeStore.pendingUpdateCount.returns(3);
+    });
+
+    it("doesn't display button to synchronize annotations if filters are applied", () => {
+      fakeStore.hasAppliedFilter.returns(true);
+      const wrapper = createComponent();
+
+      const button = wrapper.find('IconButton');
+      assert.isFalse(button.exists());
+    });
+
+    it('shows button to synchronize annotations if no filters are applied', () => {
+      const wrapper = createComponent();
+
+      const button = wrapper.find('IconButton');
```

Comment:
Same as above.

Reply:
I search now for icon=refresh, which should be more specific: `const button = wrapper.find('IconButton[icon="refresh"]');`

Thanks